subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -546,8 +546,8 @@ static int __wifi_args_to_params(size_t argc, char *argv[],
 			if (!found) {
 				print(sh, SHELL_ERROR,
```

Comment:
Shouldn't these now be `PR_ERROR`?

Reply:
The print(sh, ...) macros appear to handle a NULL `sh` value, as seen in the code

https://github.com/zephyrproject-rtos/zephyr/blob/b3e67cf2a62b01fbf9f4c0eb618cd676b4dc3072/subsys/net/l2/wifi/wifi_shell.c#L79-L86

Therefore, I have left it unchanged.
However, as @jukkar mentioned, there is no likelihood that `sh` will be `NULL`.
If this is confirmed, I will then apply the change to all places where the `print(sh, ...)` macros are used and remove those macros.

Reply:
Sure, it does handle `NULL`, but better to stick with a single macro to print.

Reply:
Ok, so I applied this commit: https://github.com/zephyrproject-rtos/zephyr/pull/68185/commits/2fe89d67e5afab360e454b37361f8f8942e3506c